src/signals/incident-management/containers/IncidentDetailPage/components/IncidentDetail/index.js

```diff
@@ -55,6 +56,20 @@ class IncidentDetail extends React.Component { // eslint-disable-line react/pref
     }
   }
 
+  static getExtraPropertyValue(answer) {
```

Comment:
Probably preferable to put logic in separate file.

Reply:
ja eens: ik heb er een apart ExtraProperties component van gemaakt